BUILDING.md

```diff
@@ -1,255 +1,49 @@
-# Building VTR #
+# Building VTR
```

Comment:
I suspect this is not intended. Please confirm. We are not supposed to modify any documentation in this PR.